mm2src/coins/utxo.rs

```diff
@@ -500,6 +504,17 @@ impl UtxoSyncStatusLoopHandle {
     }
 }
 
+/// Priority levels for UTXO fee estimation for withdrawal.
+#[derive(Clone, Debug, Deserialize)]
+pub struct UtxoFeePriorities {
```

Comment:
It seems to me UtxoFeePriorities does not exactly matches the concept.
How about
``` 
#[derive(Clone, Debug, Deserialize)]
struct UtxoTxWaitBlocks {
    pub low_priority: u8,
    pub normal_priority: u8,
    pub high_priority: u8,
}
```
In the coins file the param could have a name like `blocks_tx_to_wait` or `tx_wait_blocks`